include/experimental/__p2630_bits/submdspan_extents.hpp

```diff
@@ -52,6 +53,33 @@ template <class OffsetType, class ExtentType, class StrideType>
 struct is_strided_slice<
     strided_slice<OffsetType, ExtentType, StrideType>> : std::true_type {};
 
+// Helper for identifying valid pair like things
+template <class T, class IndexType> struct index_pair_like : std::false_type {};
```

Comment:
The feature macro for complex `get` support is called [__cpp_lib_tuple_like](https://en.cppreference.com/w/cpp/feature_test#cpp_lib_tuple_like) so maybe
```suggestion
template <class T, class IndexType> struct index_tuple_like : std::false_type {};
```

Reply:
I fixed that by adding special versions for complex.